### ISSUE TITLE
Album Art Caching

### ISSUE THREAD

**Issue by dhiachou** [icefields/Power-Ampache-2#126]
Rendering album art is currently kind of slow (depending on internet speed + server responsiveness).

It would be nice to cache the album arts locally on the phone to avoid reloading them from the server every time we access an album or a song.

**Comment by icefields** [comment]
hi,
album arts are already cached. I use both memory cache and disk cache. I reserve 5% of the app memory for image cache, which are also cached aggressively on disk to be available in offline mode.

**Comment by icefields** [comment]
I just tested the cache. 
- I verified that they are loaded in memory using the memory analizer in Android Studio.
- I checked the subdirectory that I use for caching, I can see the images there. 

As a last test I put the phone in offline mode and images still load (directly from cache).

Here's a screenshot of the directory containing the cached images on my device after a fresh installation of the app.

![image](https://github.com/icefields/Power-Ampache-2/assets/149625124/47d22b3c-c78f-4643-9d74-cd9654ac7431)
